config_ipa.go

```diff
@@ -65,12 +96,24 @@ func GetConfig() *Config {
 			ipacfg = ipa.NewIPASettingsWithSRSPrecomp(srs)
 		}
 		cfg = &IPAConfig{conf: ipacfg}
+
+		emptyHashCode, _ := hex.DecodeString("c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470")
+		values := make([][]byte, NodeWidth)
+		values[codeHashVectorPosition] = emptyHashCode[:]
+		var c1poly [NodeWidth]Fr
+		fillSuffixTreePoly(c1poly[:], values[:NodeWidth/2])
+		emptyCodeHashPoint = cfg.CommitToPoly(c1poly[:], 0)
+		emptyCodeHashFirstHalfValue = c1poly[emptyCodeHashFirstHalfIdx]
+		emptyCodeHashSecondHalfValue = c1poly[emptyCodeHashSecondHalfIdx]
```

Comment:
As part of the precomp setup, we calculate the cached point. We leverage the same methods in the VKT data structure, so things should be coherent.